tests/app/main/test_briefs.py

```diff
@@ -818,6 +818,190 @@ def test_create_new_brief_response_redirects_to_login_for_buyer(self, data_api_c
 
 
 @mock.patch("app.main.views.briefs.data_api_client")
+class TestStartBriefResponseApplication(BaseApplicationTest):
+    def setup(self):
+        super(TestStartBriefResponseApplication, self).setup()
+
+        with self.app.test_client():
+            self.login()
+
+    @mock.patch("app.main.views.briefs.is_supplier_eligible_for_brief")
+    def test_will_show_not_eligible_response_if_supplier_is_not_eligible_for_brief(
+        self, is_supplier_eligible_for_brief, data_api_client
+    ):
+        brief = api_stubs.brief(status='live', lot_slug='digital-specialists')
+        data_api_client.get_brief.return_value = brief
+        data_api_client.find_brief_responses.return_value = {
+            'briefResponses': []
+        }
+        is_supplier_eligible_for_brief.return_value = False
+
+        res = self.client.get('/suppliers/opportunities/1234/responses/start')
+        assert res.status_code == 400
+
+        doc = html.fromstring(res.get_data(as_text=True))
+        assert doc.xpath('//title')[0].text == 'Not eligible for opportunity – Digital Marketplace'
+
+    def test_start_application_contains_brief_title(self, data_api_client):
+        data_api_client.get_brief.return_value = api_stubs.brief(status='live', lot_slug='digital-specialists')
+        data_api_client.find_brief_responses.return_value = {
+            'briefResponses': []
+        }
+        res = self.client.get('/suppliers/opportunities/1234/responses/start')
+        assert res.status_code == 200
+
+        doc = html.fromstring(res.get_data(as_text=True))
+        assert doc.xpath('//h1')[0].text.strip() == "Apply for ‘I need a thing to do a thing’"
+        assert doc.xpath("//input[@class='button-save']/@value")[0] == 'Start application'
+
+    def test_start_page_is_viewable_and_has_start_button_if_no_existing_brief_response(
```

Comment:
Well spotted! I will remove the assertion for the button content from the first test as it doesn't belong in there and keep the second test (as this gives us two tests to mirror the two cases for when the button says 'start application' and where it says 'continue application')